src/index.ts

```diff
@@ -643,11 +641,11 @@ class Storyblok {
 				return resolve(response)
 			} catch (error: Error | any) {
 				if (error.response && error.status === 429) {
-					retries = retries ? retries + 1 : 0
+					retries = typeof retries === 'undefined' ? 0 : retries + 1
 
 					if (retries < this.maxRetries) {
-						console.log(`Hit rate limit. Retrying in ${retries} seconds.`)
-						await this.helpers.delay(1000 * retries)
+						console.log(`Hit rate limit. Retrying in ${this.retriesDelay/1000} seconds.`)
+						await this.helpers.delay(this.retriesDelay)
```

Comment:
The delay was previously much greater, now it will be 300 milliseconds, with this reduction, will we not have any impact on the backend?

For me it's not a problem, I don't have any test cases to check the impact of this.

Reply:
The previous limit was higher, but it was never used. Because of this bug, the delay is always 0 because it's calculated as `retries * 1000` which is always 0. So, funnily enough, this is an increase actually, I realize it just now 😅